homeTask_7/homeTask_7/Program.cs

```diff
@@ -50,6 +54,46 @@ static void Main(string[] args)
                     Console.Write($"{item.GetEmployee()}");
                     ((IReview)item).CodeReview();
                 }
+                
+            }
+            //8 task
+            Console.WriteLine("-----Sort AssigneTask by Name------");
+            office.Employees.Sort(new AssigneTaskCompare());
+            foreach (Employee item_1 in office.Employees)
+            {
+                Console.Write($"{item_1.GetEmployee()}");
+                if (item_1 is IAssigner)
+                {
+                    Console.Write(" assigner\n");
+                }
+                else Console.Write("\n");
+            }
+            Console.WriteLine("-----Sort by Name------");
+            office.Employees.Sort(new FirstNameCompare());
+            foreach (Employee item_1 in office.Employees)
+            {
+                Console.Write($"{item_1.GetEmployee()}\n");
+
```

Comment:
remove blank lines in this places but divide logical parts by blank lines
for example after this foreach

Reply:
> remove blank lines in this places but divide logical parts by blank lines for example after this foreach

Done.